### DISCUSSION TITLE
V0.7.13-beta(Web1.5.9.0)

### DISCUSSION THREAD

**Discussion by cloud-fs** [cloud-fs/cloud-fs.github.io#141] [Announcements]
V0.7.13-beta(Web1.5.9.0)

1. 根据115的新规则，修复频繁访问115网盘会被风控的问题，115的MaxQueriesPerSecond默认值和最大值限定为4。注意，请不要同时在多个同账户的CD2终端上频繁访问同一个115网盘，否则还有可能被风控
2. 修复在线更新各种不稳定的问题，当前的版本如果无法在线更新，请先手动安装新版本，后续版本更新会更加稳定
3. 修复打开目录缓存本地持久化之后的一些不稳定和死锁的bug
4. 修复115网盘直接重新登录替换后，当前正在缓存内的文件没有更新cookie的问题
5. 修复直接替换更新网盘后，网盘的下载参数和代理设置被重置的问题
6. 其它bug修复和性能改进

<hr /><em>This discussion was created from the release <a href='https://github.com/cloud-fs/cloud-fs.github.io/releases/tag/v0.7.13-beta'>V0.7.13-beta(Web1.5.9.0)</a>.</em>

**Comment by Marxpipiman** [comment]
辛苦了，目前為止使用正常
